docs/detections/rules-coverage.asciidoc

```diff
@@ -8,7 +8,7 @@
 
 The **MITRE ATT&CK® coverage** page shows which https://attack.mitre.org[MITRE ATT&CK®] adversary tactics and techniques are covered by your installed and enabled detection rules.
```

Comment:
It also includes custom rules, we should add that

Reply:
By default, we show all enabled rules - prebuilt and custom, but users can add disabled rules to the view, or change filters.

Reply:
```suggestion
The **MITRE ATT&CK® coverage** page shows which https://attack.mitre.org[MITRE ATT&CK®] adversary tactics and techniques are covered by your installed and enabled detection rules. This includes both Elastic prebuilt rules and custom rules.
```